modules/ai/cognitiveservices/main.bicep

```
@@ -0,0 +1,175 @@
+@description('The kind of Cognitive Service to create. See: https://learn.microsoft.com/en-us/azure/cognitive-services/create-account-bicep for available kinds.')
+param kind string
```

i like to have a default value wherever we can.  Would a default value of 'CognitiveServices' here work okay?

```suggestion
param kind string = 'CognitiveServices'
```

---

Mmmm... I think so.
Since it's the only `kind` that supports multiple services, it does seem like the reasonable choice here.